### PR TITLE
improve README by listing the problems of interest

### PR DESCRIPTION
**Pull request by josephsalmon**
Time to go back on benchOpt folks ;)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,3 +32,19 @@ benchopt run lasso -s sklearn -s baseline -d boston --max-samples 10 --repetitio
 ```
 
 Use `benchopt run -h` for more details about these options, or visit https://benchopt.github.io/api.html.
+
+
+
+List of optimization problems available
+-----
```
Comment (agramfort) [inline README.md:39]:
```suggestion
---------------------------------------------------
```

File: README.md [hunk 2]
```diff
@@ -32,3 +32,19 @@ benchopt run lasso -s sklearn -s baseline -d boston --max-samples 10 --repetitio
...
+
+- `lasso`: aka l1-regularized least-squares. This consists in solving the following program:
+
+```min_w (1 / (2 * n_samples)) * ||y - Xw||^2_2 + alpha * ||w||_1```
+
+-`logreg`: aka l1-regularized least-squares. This consists in solving the following program
```
Comment (agramfort) [inline README.md:45]:
```suggestion
-`logreg`: aka l1-regularized logistic regression. This consists in solving the following program
```

File: README.md [hunk 3]
```diff
@@ -32,3 +32,18 @@ benchopt run lasso -s sklearn -s baseline -d boston --max-samples 10 --repetitio
 ```
 
 Use `benchopt run -h` for more details about these options, or visit https://benchopt.github.io/api.html.
+
+
+
+List of optimization problems available
+---------------------------------------------------
```
Comment (agramfort) [inline README.md:39]: wait a sec it's a markdown file not an RST file. So use:

`### List of optimization problems available`

**Comment by josephsalmon** [comment]
@tomMoral : any feedback on that one?

**Comment by agramfort** [comment]
thx @josephsalmon

**Comment by agramfort** [comment]
thx @josephsalmon

**Pull request merged**